src/main/java/mate/academy/EventManager.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
 
+    private final List<EventListener> managedListeners = new ArrayList<>();
```

Comment:
It's better to use some thread-safe collection here. CopyOnWriteArrayList is perfect for this use case (as we expect frequent usage with rare collection modifications)